src/loophole.coffee

```diff
@@ -2,19 +2,35 @@ vm = require 'vm'
 
 exports.allowUnsafeEval = (fn) ->
   previousEval = global.eval
+  callback = -> global.eval = previousEval
```

Comment:
Naming `callback` more explicitly would be helpful, something like `restorePreviousEval`.